doc/build_the_operator_images_by_yourself_howto.md

```diff
@@ -0,0 +1,30 @@
+# Build the operator images by yourself
+
+Get the official operator image from [here](https://hub.docker.com/r/starrocks/operator/tags).
+
+### Build starrocks operator docker image
+
+Follow below instructions if you want to build your own image.
+
+```console
+DOCKER_BUILDKIT=1 docker build -t starrocks-kubernetes-operator/operator:<tag> .
+```
+
+E.g.
+
+```console
+DOCKER_BUILDKIT=1 docker build -t starrocks-kubernetes-operator/operator:latest .
+```
+
+### Publish starrocks operator docker image
+
+```console
+docker push ghcr.io/OWNER/starrocks-kubernetes-operator/operator:latest
+```
+
+E.g.
+Publish image to ghcr
+
+```console
+docker push ghcr.io/dengliu/starrocks-kubernetes-operator/operator:latest
```

Comment:
update the example with our starrocks namespace as example. remove ghcr.io, use docker hub as example.